<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_menu</class>
 <widget class="QDialog" name="main_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #05B893;</string>
  </property>
  <widget class="QPushButton" name="cash_withdraw_btn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>150</y>
     <width>431</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>401</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>10000</width>
     <height>10000</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Norwester</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: #007871;
font-size: 22px;
padding-left: 40px;
text-align: left;
 border-width: 10px;
 border-radius: 30px;</string>
   </property>
   <property name="text">
    <string>CASH WITHDRAW</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fund_transfer">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>300</y>
     <width>431</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>401</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>10000</width>
     <height>10000</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Norwester</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: #007871;
font-size: 22px;
padding-left: 40px;
text-align: left;
 border-width: 10px;
 border-radius: 30px;</string>
   </property>
   <property name="text">
    <string>FUNDS TRANSFER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show_balance_btn">
   <property name="geometry">
    <rect>
     <x>-140</x>
     <y>150</y>
     <width>431</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>401</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>10000</width>
     <height>10000</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Norwester</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: #007871;
font-size: 22px;
padding-right: 40px;
text-align: right;
 border-width: 10px;
 border-radius: 30px;</string>
   </property>
   <property name="text">
    <string>SHOW BALANCE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bill_payments_btn">
   <property name="geometry">
    <rect>
     <x>-140</x>
     <y>300</y>
     <width>431</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>401</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>10000</width>
     <height>10000</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Norwester</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: #007871;
font-size: 22px;
padding-right: 40px;
text-align: right;
 border-width: 10px;
 border-radius: 30px;</string>
   </property>
   <property name="text">
    <string>BILL PAYMENTS</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcome_label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Norwester</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 30px;
color: white;
</string>
   </property>
   <property name="text">
    <string>Welcome</string>
   </property>
  </widget>
  <widget class="QLabel" name="user_full_name">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Norwester</family>
     <pointsize>-1</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: #F54646;
font-size: 20px;
letter-spacing: 0.5em;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="username_ID">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Norwester</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #007871;
font-size: 15px;
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mainLogo">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>291</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_quit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>420</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Norwester</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #F54646;
color: white;
 border-width: 5px;
 border-radius:7px;
</string>
   </property>
   <property name="text">
    <string>QUIT</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cash_withdraw_btn</tabstop>
  <tabstop>show_balance_btn</tabstop>
  <tabstop>bill_payments_btn</tabstop>
  <tabstop>fund_transfer</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
